crates/vm/levm/bench/revm_comparison/src/levm_bench.rs

```diff
@@ -76,5 +80,5 @@ fn init_vm(db: &mut GeneralizedDatabase, nonce: u64, calldata: Bytes) -> VM {
         data: calldata,
         ..Default::default()
     });
-    VM::new(env, db, &tx, LevmCallTracer::disabled())
+    VM::new(env, db, &tx, LevmCallTracer::disabled(), VMType::L1) // TODO: Should we support this for L2?
```

Comment:
We don't want to support L2 execution with REVM
```suggestion
    VM::new(env, db, &tx, LevmCallTracer::disabled(), VMType::L1)
```